kobo/apps/project_ownership/docs/api/v2/project_ownership/invites/delete.md

```diff
@@ -0,0 +1 @@
+## Delete invite
```

Comment:
Same error issue as with the POST request. I can't make a DELETE without getting this error but it works fine locally.

<img width="1081" height="659" alt="Image" src="https://github.com/user-attachments/assets/a7fea5c6-5598-4d03-b723-35b2e861944a" />